src/main/resources/assets/bongo/lang/pl_pl.json

```diff
@@ -0,0 +1,74 @@
+{
+  "bongo.backpack": "Plecak: ",
+  "bongo.big_overlay": "Powiększ nakładkę z bingo",
+  "bongo.cmd.bp.norun": "Nie możesz uzyskać dostępu do plecaka swojej drużyny, gdy gra nie jest w trakcie trwania.",
+  "bongo.cmd.bp.noteam": "Dołącz do drużyny, aby uzyskać dostęp do jej plecaka.",
+  "bongo.cmd.create.done": " utworzył(a) nową grę.",
+  "bongo.cmd.create.less": "Nie ma wystarczająco dużo zadań w tym bingo.",
+  "bongo.cmd.create.notfound": "Nie udało się znaleźć tej definicji gry.",
```

Comment:
Could you please provide me context for this sentence? I tried to get it on my own, but I failed.

Reply:
What sentence exactly? There are 4 lines highlighted

Reply:
That's GitHub's fault. I mean line 8.

Reply:
It is sent as command feedback when you give an invalid ResourceLocation to `/bingo create` that does not exist in any data-pack.

Reply:
Oh I see. Give me a while, I'm going to change the translated sentence a bit.

Reply:
Done.